src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -364,6 +368,15 @@ const props = defineProps({
     default: true,
   },
 
+  /**
+   * Controls whether to show line numbers. **Default: `true`**.
+   */
+  showLineNumbers: {
```

Comment:
nit: when we went through the reskin effort last year we tried to get into the practice of removing prefixes from props like “has” and “is” and “show”

Could we rename the prop `lineNumbers`?

Reply:
I would have named it as `line-numbers` if not for the existing `show-line-number-links` and `show-copy-button`…removing the `show` prefix feels weird now.

Reply:
oof yep 🫣